src/current/v24.3/sso-sql.md

```diff
@@ -38,19 +38,21 @@ The page describes how to configure a cluster for cluster single sign-on using J
 
 You must configure the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) in the following table to enable JWT authentication to your cluster. Refer to the [Update your cluster settings](#update-your-cluster-settings) section to configure your cluster settings.
 
-| Cluster Setting | Description
-|-----------------|------
-| `server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation.
-| `server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. If `server.jwt_authentication.jwks_auto_fetch.enabled` is `true`, there is no need to set `server.jwt_authentication.jwks`.
-| `server.jwt_authentication.jwks_auto_fetch.enabled` | If `true`, public signing keys are automatically fetched from the issuer and there is no need to set `server.jwt_authentication.jwks`. Defaults to `false`.
-| `server.jwt_authentication.issuers` | A list of accepted token issuers; must include your IdP.
-| `server.jwt_authentication.audience` | This must match `server.oidc_authentication.client_id`; refer to [Single Sign-on (SSO) for DB Console](sso-db-console.html).
-| `server.jwt_authentication.claim` | Which JWT field will be used to determine the user identity in CockroachDB; normally set either to `email`, or `sub` (subject).
-| `server.oidc_authentication.generate_cluster_sso_token.enabled` | Enables token generation; must be set to `true`.
-|`server.oidc_authentication.generate_cluster_sso_token.use_token`| Selects which part of the received OIDC credentials to display.
-|`server.identity_map.configuration`| Takes an [Identity Map configuration](#identity-map-configuration).
-| `server.sql_host` | This display value informs users the host for their SQL connections. Default: `localhost`.
-| `server.sql_port` | This display value informs users the port for their SQL connections. Default: `26257`.
+Cluster Setting | Description
+----------------|------
+`server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation.
+`server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. If `server.jwt_authentication.jwks_auto_fetch.enabled` is `true`, there is no need to set `server.jwt_authentication.jwks`.
+`server.jwt_authentication.jwks_auto_fetch.enabled` | If `true`, public signing keys are automatically fetched from the issuer and there is no need to set `server.jwt_authentication.jwks`. Defaults to `false`.
+`server.jwt_authentication.issuers.configuration` | A list of accepted token issuers; must include your IdP. Can be any of the following: <ul><li>A string representing a valid issuer URL</li><li>A string that contains a JSON array of issuer URLs</li><li>A string that contains a JSON map of issuer URLs</li></ul>The format is detected automatically.
```

Comment:
Should we mention that it is an alias to the previous `server.jwt_authentication.issuers` setting?